src/NLog.Web.AspNetCore/README.md

```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform via extension methods for ILoggingProvider, IWebHostBuilder, and IHostBuilder.
+- Log all properties of HttpContext and IHostEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore) just by changing the NLog.config file.
+- Capable to log to many destinations via [targets](https://nlog-project.org/config/?tab=targets).
+- Supports common [layouts](https://nlog-project.org/config/?tab=layouts), such as CVS, JSON, W3C ELF, and XML.
```

Comment:
"Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog Layouts"

Reply:
Completed